test/fixtures/schema.sql

```diff
@@ -1819,8 +1819,8 @@ create table v1.parents (
 );
 
 create table v1.children (
-  id       serial primary key
```

Comment:
Next to ordering of rows, this is something else that is not restored after rollback. Tests should be made more independent of auto-generated IDs in the future, I guess.